d2bs/kolbot/default.dbj

```diff
@@ -63,6 +65,18 @@ function main() {
 
 	// don't load default for mules
 	if (getScript("D2BotMule.dbj")) {
+		while(true) {
```

Comment:
nit: formatting

Reply:
linted. Additionally, I linted the D2BotSoloLevelingEntry.dbj.